dist/10-10EZ-schema.json

```diff
@@ -1836,6 +1748,11 @@
     "isSpanishHispanicLatino",
     "veteranAddress",
     "isMedicaidEligible",
-    "isEssentialAcaCoverage"
+    "isEssentialAcaCoverage",
+    "vaMedicalFacility",
+    "lastServiceBranch",
+    "lastEntryDate",
+    "lastDischargeDate",
+    "dischargeType"
```

Comment:
"lastServiceBranch",
 "lastEntryDate",
 "lastDischargeDate",
 "dischargeType"

all these fields should not be required in the schema because they are not required for short form users

Reply:
Removed